pages/[user].tsx

```diff
@@ -50,21 +54,13 @@ export default function User(props): User {
 }
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
-  const user = await prisma.user.findFirst({
-    where: {
-      username: context.query.user.toLowerCase(),
-    },
-    select: {
-      id: true,
-      username: true,
-      email: true,
-      name: true,
-      bio: true,
-      avatar: true,
-      eventTypes: true,
-    },
-  });
 
+  const user = await whereAndSelect(prisma.user.findFirst, {
+      username: context.query.user.toLowerCase(),
+    }, [
```

Comment:
Codacy has a fix for the issue: [Delete `·[`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
    },
```